recipe/meta.yaml

```diff
@@ -9,24 +9,38 @@ source:
   sha256: 502ad98707319f4a51fa2ca1c677bd659008d27ded9f6380c79e8932e38dcdf2
 
 build:
-  number: 1
-  script: python setup.py install
+  number: 2
   skip: True  # [py>=35]
 
 requirements:
-  build:
-    - python
-  run:
+  host:
     - python
+    - pip
 
 test:
   imports:
     - backports
     - backports.ssl_match_hostname
 
+outputs:
+  - name: ssl_match_hostname
+    build:
+      script: python -m pip install --no-deps --ignore-installed .
+    requirements:
+      - python
+      - backports
+      - ipaddress
+
+  - name: backports.ssl_match_hostname
```

Comment:
Should we interchange the names?

Reply:
That would make more sense, but I am not sure it matters

Reply:
Agree it would be good to interchange these. Would like to move away from `ssl_match_hostname` being the name and towards `backports.ssl_match_hostname` being the name.